tests/unit/testplan/testing/multitest/driver/test_tcp_server.py

```diff
@@ -38,6 +39,46 @@ def tcp_client(tcp_server, runpath_module):
         yield client
 
 
+@pytest.fixture(scope="module")
+def tcp_server_port_aliased(runpath_module):
+    """Start and yield a TCP server driver using port aliasing."""
+    env = Environment()
+
+    def get_mocked_port_name():
+        for port in range(2000, 65535):
+            try:
+                return socket.getservbyport(port)
+            except:
+                pass
+
+    mocked_port = get_mocked_port_name()
+
+    server = TCPServer(
+        name="server_port_aliased",
+        host="localhost",
+        port=mocked_port,
+        runpath=runpath_module,
+    )
+    env.add(server)
+
+    with server:
+        yield server
+
+
+@pytest.fixture(scope="module")
+def tcp_client_port_aliased(tcp_server_port_aliased, runpath_module):
+    """Start and yield a TCP client driver using port aliasing."""
+    client = TCPClient(
+        name="client_port_aliased",
+        host=tcp_server_port_aliased.host,
+        port=tcp_server_port_aliased.port,
```

Comment:
here the port is for sure numeric so the new logic is not excercised

Reply:
Because the `tcp_client_port_aliased` uses the `tcp_server_port_aliased `fixture. there is a dependency (chain) so it is the actual test `test_send_receive_with_port_aliasing` which asserts it.

Reply:
tcp_server_port_aliased.port always return an int as the port is already resolved during startup time, so the client will do not need to run the resolution.